This patch makes SDK_DelayAtLeastUs() a weak symbol so that it can be overridden by Mbed OS
for some MCUs.

diff --git a/drivers/common/fsl_common_arm.c b/drivers/common/fsl_common_arm.c
index dfa1e67..22f93c3 100644
--- a/drivers/common/fsl_common_arm.c
+++ b/drivers/common/fsl_common_arm.c
@@ -214,7 +214,7 @@ static void DelayLoop(uint32_t count)
  * @param delayTime_us  Delay time in unit of microsecond.
  * @param coreClock_Hz  Core clock frequency with Hz.
  */
-void SDK_DelayAtLeastUs(uint32_t delayTime_us, uint32_t coreClock_Hz)
+__attribute__((weak)) void SDK_DelayAtLeastUs(uint32_t delayTime_us, uint32_t coreClock_Hz)
 {
     uint64_t count;
 
